stacks/JAVA/Balanced_Paranthesis.java

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
Please rename the folder's name to `java`, and rename the file's name to `balanced_paranthesis.java`

Reply:
Done with this just now. can you check now please?